meilisearch/index.py

```diff
@@ -592,6 +592,33 @@ def update_documents_json(
         """
         return self.update_documents_raw(str_documents, primary_key, "application/json")
 
+    def update_documents_csv(
+        self,
+        str_documents: str,
+        primary_key: Optional[str] = None,
+    ) -> TaskInfo:
+        """Update documents as a csv string in the index.
+
+        Parameters
+        ----------
+        str_documents:
+            String of document from a CSV file.
+        primary_key (optional):
+            The primary-key used in index. Ignored if already set up
+
+        Returns
+        -------
+        task_info:
+            TaskInfo instance containing information about a task to track the progress of an asynchronous process.
+            https://docs.meilisearch.com/reference/api/tasks.html#get-one-task
+
+        Raises
+        ------
+        MeiliSearchApiError
+            An error containing details about why Meilisearch can't process your request. Meilisearch error codes are described here: https://docs.meilisearch.com/errors/#meilisearch-errors
+        """
+        return self.update_documents_raw(str_documents, primary_key, "application/csv")
```

Comment:
This should have `text/csv` instead of `application/csv`

```suggestion
        return self.update_documents_raw(str_documents, primary_key, "text/csv")
````